tests/backup/backup_helper.go

```diff
@@ -2027,3 +2029,26 @@ func MapToKeyValueString(m map[string]string) string {
 	}
 	return strings.Join(pairs, ",")
 }
+
+// VerifyLicenseConsumedCount verifies the consumed license count for px-backup
+func VerifyLicenseConsumedCount(ctx context.Context, OrgId string, expectedLicenseConsumedCount int64) error {
+	licenseInspectRequestObject := &api.LicenseInspectRequest{
+		OrgId: OrgId,
+	}
+	licenseCountCheck := func() (interface{}, bool, error) {
+		licenseInspectResponse, err := Inst().Backup.InspectLicense(ctx, licenseInspectRequestObject)
+		if err != nil {
+			return "", false, err
+		}
+		licenseResponseInfoFeatureInfo := licenseInspectResponse.GetLicenseRespInfo().GetFeatureInfo()
+		if licenseResponseInfoFeatureInfo[0].Consumed == expectedLicenseConsumedCount {
+			return "", false, nil
+		}
+		return "", true, fmt.Errorf("actual license count:%v ,expected license count: %v", licenseInspectResponse.GetLicenseRespInfo().GetFeatureInfo()[0].Consumed, expectedLicenseConsumedCount)
```

Comment:
nit: The comma placement can be corrected

Reply:
done